pkg/notifications/notifications_integration_test.go

```diff
@@ -93,6 +93,84 @@ func TestNotificationMutationDestination(t *testing.T) {
 	require.NotNil(t, deleteResult)
 }
 
+func TestNotificationMutationDestinationByName(t *testing.T) {
```

Comment:
We are just testing data source, not the update mutation.
I would suggest to both rename the test (to, e.g., `TestNotificationGetDestinationByName`), as well as remove  the irrelevant parts from it (see comments below)